File-Formats/PDF/Working-with-Annotations.md

```diff
@@ -1033,6 +1033,324 @@ else
 
 {% endtabs %}
 
+## Flattening annotations without calling save method 
+
+Annotations can be flattened by removing the existing annotation and replacing it with graphic objects that would resemble the annotation and cannot be edited.
+
+This can be achieved by enabling the FlattenAnnotation method. Please refer to the sample for flattening all the annotations in the PDF document without calling the save method
```

Comment:
This can be achieved by calling the FlattenAnnotation method. - change as like

Reply:
Changed the name enabling->calling